video-browser-api/src/main/java/no/sikt/nva/vms/browser/model/Panopto.java

```diff
@@ -0,0 +1,66 @@
+package no.sikt.nva.vms.browser.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public class Panopto {
+
+    private String instanceEndpoint;
+    private String username;
+    private String password;
+    private String identityProviderInstanceName;
+    private String serverFqdn;
+    private String applicationKey;
+    private String clientId;
+    private String clientSecret;
+
```

Comment:
Only default constructor and no setters?

Reply:
Ja, det er en pojo som blir brukt kun for henting av configen

Reply:
Har ikke lagt merke at intellij klager på at den mangler konstruktøren, endrer dette både her og andre steder